runtime/devnet/Cargo.toml

```diff
@@ -0,0 +1,198 @@
+[package]
+name = "devnet-runtime"
+version = "0.1.0"
+authors.workspace = true
+description = "A new Cumulus FRAME-based Substrate Runtime, ready for hacking together a parachain."
```

Comment:
can we update the information to metaquity?

Reply:
Updated